extensions/nodes/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "nodes",
+  "title": "Nodes",
+  "description": "Browse Solana Validator Stats from Stakewiz",
+  "icon": "extension-logo.png",
+  "author": "jstnw10",
+  "categories": [
+    "Data",
+    "Developer Tools"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "menuBarStats",
+      "title": "Solana Validator",
+      "description": "Displays Validator Stats",
+      "mode": "menu-bar",
+      "interval": "1m"
```

Comment:
Since this has a static title we should maybe remove the internal since it updates when the user clicks

Reply:
Gotcha, right. Was looking to show some toast if a certain value changes in the future, didn't implement it yet (and don't know if that is entirely possible). Will remove for now.